libraries/integration/Microsoft.Bot.Builder.Integration.AspNet.Core/BotFrameworkHttpClient.cs

```diff
@@ -211,7 +211,15 @@ protected virtual async Task<AppCredentials> BuildCredentialsAsync(string appId,
 
         private static T GetBodyContent<T>(string content)
         {
-            return JsonConvert.DeserializeObject<T>(content);
+            try
+            {
+                return JsonConvert.DeserializeObject<T>(content);
+            }
+            catch (JsonException)
+            {
+                // This will only happen when the skill didn't return valid json in the content (e.g. when the status code is 500 or there's a bug in the skill)
+                return default;
```

Comment:
@gabog  Can we log the content, and/or exception when this happens?  Will this be discoverable?

Reply:
Hi Eric, I discussed this with PVA and content may contain confidential data and we can't make the decision to log it at the framework level. They are OK with just getting the status code back at this point (and we can't force the content into the body because we don't know what type T is).

Reply:
@gabog can we at least log that an exception has occurred?  I'm just afraid that swallowing exceptions returned from the skill will make it extremely difficult to determine why a skill does not return the expected value.  Especially for developers who do not own the skill code.